lib/index.js

```diff
@@ -22,13 +22,13 @@ const checkEngine = (target, npmVer, nodeVer, force = false) => {
 
 const isMusl = (file) => file.includes('libc.musl-') || file.includes('ld-musl-')
 
-const checkPlatform = (target, force = false) => {
+const checkPlatform = (target, force = false, environment = {}) => {
   if (force) {
     return
   }
 
-  const platform = process.platform
-  const arch = process.arch
+  const platform = environment && environment.os ? environment.os : process.platform
```

Comment:
Since we're defaulting environment to `{}` here we don't need the `environment &&` guard.

Reply:
@wraithgar Callers of this function may still pass `null` or `undefined` for this argument while the default value is `{}`. How about leaving the `environment &&` guard as it is and changing the default value to `undefined`?

Reply:
@wraithgar Okay, that's make sense! Let's prioritize the consistency of the whole codebase. [76b0df6](https://github.com/npm/npm-install-checks/pull/65/commits/76b0df674da9499660e1b72eb41a05dbfefaf1e3)